examples/complete/variables.tf

```diff
@@ -61,6 +61,11 @@ variable "lifecycle_rules" {
   description = "A list of lifecycle rules."
 }
 
+variable "replication_rules" {
+  default     = []
+  description = "S3 replication rules"
+}
```

Comment:
nit: we could add the `s3_` prefix to this variable in `examples/complete` to be more consistent with the newly-added variables in root module.